### PR TITLE
2023-10-02 CSE Release Note

### PR DESCRIPTION
**Pull request by pkazmir-sumo**
CSE Release Note for 2023-10-02
Includes MITRE ATTACK Threat Coverage Explorer

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by jpipkin1 — APPROVED** [review]
Looks good, @pkazmir-sumo.

**Comment by jpipkin1** [comment]
@kimsauce, in the status check, we're getting spellcheck errors on files not in this PR. I'll submit a PR to fix those errors.

**Comment by jpipkin1** [comment]
@kimsauce, I temporarily turned off the spellcheck step so that this PR could merge.

File: blog-cse/2023-10-02-application-update.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+We are excited to announce a new feature in Cloud SIEM Enterprise (CSE), the MITRE ATT&amp;CK&reg; Threat Coverage Explorer. This interactive tool gives you the ability to see how Rules, Signals, and log sources map to adversary actions using the [MITRE ATT&amp;CK&reg; Matrix for Enterprise](https://attack.mitre.org/).
+
+<img src={useBaseUrl('img/release-notes/cse/MITRE-Explorer.png')} alt="MITRE ATT&amp;CK&reg; Threat Coverage Explorer"/>
+
+The MITRE Explorer can be used to identify gaps in coverage and understand the impact of specific log sources and Rules to the overall threat coverage and value of CSE.
+
+The tool can be accessed in the Content Menu. It supports three different views:
+* Recent Activity - Your environment's actual coverage (Rules that generated Signals) over the past six months
+* All Community Activity - All CSE customers' anonymized and aggregated coverage over the past six months
+* Theoretical Coverage - Potential coverage if all rules are enabled and all log sources are connected
```
Comment (jpipkin1) [inline blog-cse/2023-10-02-application-update.md:27]:
```suggestion
* Recent Activity - Your environment's actual coverage (Rules that generated Signals) over the past six months
* All Community Activity - All CSE customers' anonymized and aggregated coverage over the past six months.
* Theoretical Coverage - Potential coverage if all rules are enabled and all log sources are connected.
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**